besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1942,9 +1942,14 @@ private PrivacyParameters privacyParameters() {
         throw new ParameterException(
             commandLine, String.format("%s %s", "Checkpoint sync", errorSuffix));
       }
-      if (getDataStorageConfiguration().getDataStorageFormat().equals(DataStorageFormat.BONSAI)) {
+      if (getDataStorageConfiguration().getDataStorageFormat() == DataStorageFormat.BONSAI) {
```

Comment:
is it needed ?

Reply:
I'm not sure why that got changed. I'll revert it to the previous code.